cmake/SSGCommon.cmake

```diff
@@ -261,6 +261,12 @@ macro(ssg_build_ansible_playbooks PRODUCT)
         NAME "${PRODUCT}-ansible-playbooks-generated-for-all-rules"
         COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/tests/ansible_playbooks_generated_for_all_rules.py" --build-dir "${CMAKE_BINARY_DIR}" --product "${PRODUCT}"
     )
+    if("${PRODUCT}" MATCHES "rhel")
+        add_test(
+            NAME "${PRODUCT}-ansible-assert-playbooks-schema"
+            COMMAND sh env "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/tests/assert_ansible_schema.py" "${CMAKE_BINARY_DIR}/${PRODUCT}/playbooks/all/*"
```

Comment:
What is `sh env`?

Reply:
If one wants to use wildcards in commands, one needs the shell to expand wildcards.